src/connection.ts

```diff
@@ -157,6 +157,23 @@ abstract class Connection extends EventEmitter {
   getCurrentNetwork(): string {
     return this.network;
   }
+
+  startControlHandlers(options?: any) {
```

Comment:
The old [src/wallet.js:setConnection](https://github.com/HathorNetwork/hathor-wallet-lib/blob/25b81a822726f811979edd54a10ba5182b4f2db4/src/wallet.js#L2910-L2918) method added a metrics listener for the `dashboard` event to update the network height and a listener for `subscribe_address` which simply throws an error if the call was unsuccessful.

I called these 2 listeners "control handlers" and we need to start them when connecting with a fullnode websocket.
But since the method should also be present on the `Connection` class I added it as a no-op, but if you check the [src/new/connection.ts](https://github.com/HathorNetwork/hathor-wallet-lib/blob/feat/new-storage-scheme/src/new/connection.ts#L71-L74) you'll see that the method does add the listeners.